pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -331,6 +331,18 @@ func testPodDevices(t *testing.T, dataDir, deviceName string, allDevices bool) {
 	assert.Equal(t, 1, len(blkInitCont.VolumeDevices))
 	blkMetaInitCont = deployment.Spec.Template.Spec.InitContainers[8]
 	assert.Equal(t, 1, len(blkMetaInitCont.VolumeDevices))
+
+	// // Test tune Fast settings when OSD on PVC
+	osdProp.tuneFastDeviceClass = true
+	deployment, err = c.makeDeployment(osdProp, osd, dataPathMap)
+	assert.NoError(t, err)
+	assert.Equal(t, 30, len(deployment.Spec.Template.Spec.Containers[0].Args))
```

Comment:
The number of arguments is rather fragile. Anytime we add an argument in the future, these tests will have to be updated. What if we check for the presence of the expected argument by name? Then we would be insulated from argument count changing. However, maybe it's not worth the extra complexity in the test...

Reply:
okay. using `assert.Contains` to check the presence.